README.md

```diff
@@ -20,19 +22,29 @@ Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app
 
 ![Web App Log Stream](resources/WebApp_LogStream.png)
 
+## Configuring Zipdeploy
+
 * How to configure zip deployment:
-  * If you set the deployment source of your web app to “None”, the deploy command will Zip the contents of a selected folder and upload the Zip file to Azure. You might want to use the following settings to customize which files to include/exclude:
+  * If you set the deployment source of your web app to “None” (the default source on app creation), the deploy command will Zip the contents of a selected folder and upload the Zip file to Azure.
+
+  * VS Code will prompt on deploy if you would like to configure your project for faster deployment.  If you click "Yes", your project's settings will be changed in the following:
+    * 'appService.zipIgnorePattern' will include build artifacts to be ignored for zipping.  These will now be built on the server by running the appropriate build command.
+    * A '.deployment' file will be created in the root of the project.  This file configures 'SCM_DO_BUILD_DURING_DEPLOYMENT=true,' enabling build on deploy.
```

Comment:
I would remove the single quotes from SCM_DO_BUILD_DURING_DEPLOYMENT=true, it's confusing, especially with the comma inside it.